.github/workflows/integration-package-release.yaml

```diff
@@ -64,9 +64,13 @@ jobs:
           name: ${{ needs.build-pypi-dists.outputs.PACKAGE_NAME}}-pypi-dists
           path: "./dist"
 
+      - name: Determine secret key
+        run: |
+          PACKAGE_NAME=${{ needs.build-pypi-dists.outputs.PACKAGE_NAME }}
+          echo "SECRET_KEY=$(echo $PACKAGE_NAME | awk -F'-' '{ print "PYPI_" toupper($2) "_API_TOKEN" }')" >> $GITHUB_ENV
+
       - name: Publish distribution to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
-          repository-url: https://test.pypi.org/legacy/
-          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          password: ${{ secrets[env.SECRET_KEY] }}
```

Comment:
whoa I didn't know you could do this

Reply:
If the internet is to be believed, you can, but we'll see...